vcf_stuff/filtering/annotate_somatic_vcf.smk

```diff
@@ -63,6 +65,7 @@ rule all:
 #   Remove DP<25 & AF<5% in tricky regions: gc15, gc70to75, gc75to80, gc80to85, gc85, low_complexity_51to200bp, low_complexity_gt200bp, non-GIAB confident, unless coding in cancer genes
 
 
+# Filters HMF hotspots file from 17,875 in total down to 10,209 HMF variants (Jul2022)
 rule prep_hmf_hotspots:
```

Comment:
Not relevant to the review at all but do you have a sense why we pre-filter the HMF hotspots to a subset? Somewhat worried that removes regions that were masked in the bcbio reference but are now present and mappable in DRAGEN.